src/test/java/io/github/deansg/jeocodio/IntegrationTest.java

```diff
@@ -19,12 +23,28 @@ public void setUp() {
     @Disabled("Should be used for local testing with a real API key as an environment variable")
     @Test
     public void testGeocodingE2E() throws ExecutionException, InterruptedException {
-         var q = "1109 N Highland St, Arlington VA";
+        var q = "1109 N Highland St, Arlington VA";
 
         var future = geocodioClient.geocodeAsync(q);
 
         var result = future.get();
         Assertions.assertNotNull(result);
         Assertions.assertEquals(1, result.results().size());
     }
+
+    @Disabled("Should be used for local testing with a real API key as an environment variable")
+    @Test
+    public void testGeocodingE2EWarnings() throws ExecutionException, InterruptedException {
+        // test warnings
+        var geocodingRequest = GeocodingRequestBuilder.builder()
+                .q("Road X, Hot Springs, AR")
+                .fields(List.of("acs-social"))
+                .limit(1)
+                .build();
+        var result = geocodioClient.geocodeAsync(geocodingRequest).get();
```

Comment:
please add blank lines that separate the 3 stages of the test (arrange/act/assert) as done in the other test in this file

Reply:
ok